<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table1</class>
 <widget class="QWidget" name="table1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>713</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>938</width>
    <height>713</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>938</width>
    <height>713</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>711</height>
    </rect>
   </property>
   <property name="title">
    <string>Dynamic solution process</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>679</width>
        <height>679</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="stack_table">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>320</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>CPU Status </string>
   </property>
   <widget class="QLabel" name="cpu_statu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>cpu_statu</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="speed_Slider">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>500</y>
     <width>22</width>
     <height>191</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>480</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SLOW</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>690</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>FAST</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="status_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="step_num">
      <property name="text">
       <string>搜索步数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="solution_lable">
      <property name="text">
       <string>解的数量</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>120</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Console</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="start_btn">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="step_btn">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Step</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="next_slt_btn">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Next Solution</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
